template/README.md

```diff
@@ -11,72 +11,63 @@ Documentation on [parent repository](https://github.com/fredrikhgrelland/vagrant
 ## Customizing and using the vagrant box
 The vagrant box ships with a default startup scheme. It will run an ansible playbook to start all services.
 You may change the hashistack configuration or add aditional pre and post steps to the startup procedure to match your needs.
+Detailed documentation in [vagrant/conf/README.md]()
 
-### Overriding and extending the configuration of the hashistack
-
-- consul [vagrant/conf/hashistack/consul/99-override.hcl](vagrant/conf/hashistack/consul/99-override.hcl)
-- nomad [vagrant/conf/hashistack/nomad/99-override.hcl](vagrant/conf/hashistack/nomad/99-override.hcl)
-- vault [vagrant/conf/hashistack/vault/99-override.hcl](vagrant/conf/hashistack/vault/99-override.hcl)
+#### Nomad ACLs
 
-You may edit the `99-override.hcl` or add your own.
-Any valid configuration added to these directories will be added to their respective services' configuration, in lexical order.
+| default   | environment variable  |  value  |
+|:---------:|:----------------------|:-------:|
+|           | nomad_acl             |  true   |
+|           | TF_VAR_nomad_acl      |  true   |
+| x         | nomad_acl             |  false  |
```

Comment:
Just a suggestion
```suggestion
| true         | nomad_acl             |  [true, false]  |
```

Reply:
Tried it, but it was confusing in plain MD